playwright.config.ts

```diff
@@ -11,8 +11,23 @@ export default defineConfig({
   // TODO hardcoded values should be extracted to ENVs
   retries: process.env.CI ? 2 : 0,
   workers: process.env.CI ? 2 : 2,
-  reporter: process.env.CI ? "blob" : "html",
-  timeout: 60000 ,
+  reporter: [
+    [process.env.CI ? "blob" : "html"],
+    [process.env.CI ? "github" : "list"],
+    [
+      "playwright-testmo-reporter",
```

Comment:
will it run locally when I run test manually on my machine? do I need testmo reporter then?

Reply:
why generate results then it they won't be used at all?